src/faction.h

```diff
@@ -83,6 +83,7 @@ class faction_template
         std::string currency; // itype_id of the faction currency
         std::map<std::string, std::bitset<npc_factions::rel_types>> relations;
         std::string mon_faction; // mon_faction_id of the monster faction; defaults to human
+        std::map<int, std::string> epilogues;
```

Comment:
Please use `translation` for automatical text style checking.

Reply:
Should be resolved as I decided to use text snippets functionality to store epilogue texts.